### DISCUSSION TITLE
Consider moving SystemExecutioner to kiwi

### DISCUSSION THREAD

**Discussion by sleberknight** [kiwiproject/dropwizard-service-utilities#253] [General]
The `SystemExecutioner` class and the supporting `ExecutionStrategy` interface and `ExecutionStrategies` utility class are not specific to Dropwizard. Maybe these should be moved into kiwi, since they're more generic utilities that could be used in any environment.

**Comment by chrisrohr** [comment]
Sure, sounds reasonable.

**Comment by sleberknight** [comment]
This will be fixed by the following:

- https://github.com/kiwiproject/kiwi/issues/1421
- https://github.com/kiwiproject/dropwizard-service-utilities/issues/673
- https://github.com/kiwiproject/dropwizard-service-utilities/issues/674
- https://github.com/kiwiproject/dropwizard-service-utilities/issues/675
